submissions/kovalov/oop/index.js

```diff
@@ -0,0 +1,59 @@
+class Inhabitant {
+  constructor(species, name, gender, legs, hands, saying) {
+    this.species = species;
+    this.name = name;
+    this.gender = gender;
+    this.legs = legs;
+    this.hands = hands;
+    this.saying = saying;
+    this._properties = [
+      'species',
+      'name',
+      'gender',
+      'legs',
+      'hands',
+      'saying',
+    ];
+  }
+  set properties(prop) {
```

Comment:
Is this `set` property used somewhere?

Reply:
Have added it to have ability to add some props in future and just wanted to practice setters/getters:)